### ISSUE TITLE
console command breaks

### ISSUE THREAD

**Issue by billythekid** [mmikkel/Reasons-Craft3#1]
Hi Mats, 

Thanks so much for porting this. I've found a couple of issues locally which I was going to PR but I'm not 100% sure if my "fix" has side-effects I've not gone over.

If you run `./craft project-config/sync --force` in terminal the plugin throws an error about `getBodyParam()` not existing which is true, the console version of the Craft app Request class doesn't have this method.

I "fixed" this by simply wrapping the call to it which is within the `EVENT_AFTER_SAVE_FIELD_LAYOUT` handler, in a check:
```
          if (!Craft::$app->getRequest()->getIsConsoleRequest())
          {
            // getBodyParam doesn't exist in console commands such as ./craft project-config/sync -- force
            $conditionals = Craft::$app->getRequest()->getBodyParam('_reasons', null);
          }
```
which meant the rest of the code ran through with it a null value (so it just returns) but obviously that means the thing that should be happening isn't happening so not sure if that's an issue, hence no PR for this. I think it's OK because you're attempting to handle save field events from the dashboard in here but I don't know if syncing project config with the force flag will mean the reasons stuff gets lost?

I'll open a different issue for the other thing I've found.

Cheers.

**Comment by mmikkel** [comment]
@billythekid Good catch, thanks! I'll get a fix out ASAP :)

**Comment by mmikkel** [comment]
Should be resolved in [Reasons 2.1.1](https://github.com/mmikkel/Reasons-Craft3/releases/tag/2.1.1).
